core/src/main/java/org/elasticsearch/index/translog/Translog.java

```diff
@@ -112,7 +110,7 @@
 
     static final Pattern PARSE_STRICT_ID_PATTERN = Pattern.compile("^" + TRANSLOG_FILE_PREFIX + "(\\d+)(\\.tlog)$");
 
-    private final List<ImmutableTranslogReader> recoveredTranslogs;
+    private final List<ImmutableTranslogReader> readers = new ArrayList<>();
```

Comment:
Can we add a comment here, that the readers are in sorted (by translog generation) order?